app/components/screens/NoteScreen.tsx

```diff
@@ -32,7 +33,6 @@ async function loadHtmlFileForAndroid() {
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: "#fff",
```

Comment:
afaik this is important for the loading screen, but can be made dynamic based on the mode

Reply:
i think this is inherited from the underlying view but if you can show me a screenshot of what you mean i can take a closer look